### PR TITLE
Implement help command and user guide fixes.

### PR DESCRIPTION
**Pull request by timleow**
Add a quick guide to help command with 3 sections. It is a condensed version of the user guide.

**General Commands**, **Project Commands**, **Employee Commands**.

image

image

Also fix user guide, replacing `person`s with `employee`s and `pr/` prefix to avoid conflict with `p/` for phone number.

### CODE REVIEW

**Review by Chandan8186 — CHANGES_REQUESTED** [review]
LGTM, but seems like there are conflicts to be resolved.

**Review by Chandan8186 — CHANGES_REQUESTED** [review]
There seem to be some differences in some of the command formats when compared with the implementation by @antonTan96, specifically how the `e/` prefix to assign employee to project is used (more details in [his PR](https://github.com/AY2324S1-CS2103T-T08-3/tp/pull/94)). Also, it seems that Anton has changed the `e/` prefix for adding email, to `em/` to avoid ambiguity.

**Review by Chandan8186 — APPROVED** [review]
LGTM!

**Review by aslam341 — APPROVED** [review]
LGTM. I understand that codecov fails due to added UI elements, so I am okay with the merge.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Format: `findE KEYWORD [MORE_KEYWORDS]`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
 * Only the name is searched.
 * Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
+* employees matching at least one keyword will be returned (i.e. `OR` search).
```
Comment (aslam341) [inline docs/UserGuide.md:136]: Minor grammatical error, but `employees` should be changed to `Employees` in the next commit.

**Pull request merged**